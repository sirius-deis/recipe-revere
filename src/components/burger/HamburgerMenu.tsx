import { FC } from "react";
import styles from "./HamburgerMenu.module.css";

const HamburgerMenu: FC = () => {
  return (
    <nav className={styles.navigation}>
      <span></span>
      <span></span>
      <span></span>
    </nav>
  );
};

export default HamburgerMenu;
