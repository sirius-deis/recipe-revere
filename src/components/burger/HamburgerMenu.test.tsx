import { render } from "@testing-library/react";
import HamburgerMenu from "./HamburgerMenu";

describe("HamburgerMenu component", () => {
  it("should match snapshot", () => {
    const { container } = render(<HamburgerMenu />);
    expect(container).toMatchSnapshot();
  });
});
