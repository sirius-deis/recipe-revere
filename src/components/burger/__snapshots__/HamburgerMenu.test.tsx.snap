// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HamburgerMenu component should match snapshot 1`] = `
<div>
  <nav
    class="navigation"
  >
    <span />
    <span />
    <span />
  </nav>
</div>
`;
